src/testdir/test_vim9_disassemble.vim

```diff
@@ -3105,26 +3105,26 @@ def Test_disassemble_interface_static_member()
     '11 STORE $0\_s*' ..
     '12 RETURN void\_s*',
     g:instr1)
-  assert_match('<SNR>\d*_F2\_s*' ..
-    'var x: number\_s*' ..
-    'x = o.s_var\_s*' ..
-    '0 LOAD arg\[-1\]\_s*' ..
-    '1 OBJ_MEMBER 0 \[STATIC\]\_s*' ..
-    '2 STORE $0\_s*' ..
-    'x = o.o_var\_s*' ..
-    '3 LOAD arg\[-1\]\_s*' ..
-    '4 OBJ_MEMBER 0\_s*' ..
-    '5 STORE $0\_s*' ..
-    'x = o.s_var2\_s*' ..
-    '6 LOAD arg\[-1\]\_s*' ..
-    '7 OBJ_MEMBER 1 \[STATIC\]\_s*' ..
-    ' 8 STORE $0\_s*' ..
-    'x = o.o_var2\_s*' ..
-    '9 LOAD arg\[-1\]\_s*' ..
-    '10 OBJ_MEMBER 1\_s*' ..
-    '11 STORE $0\_s*' ..
-    '12 RETURN void',
-    g:instr2)
+  #assert_match('<SNR>\d*_F2\_s*' ..
+  #  'var x: number\_s*' ..
+  #  'x = o.s_var\_s*' ..
+  #  '0 LOAD arg\[-1\]\_s*' ..
+  #  '1 OBJ_MEMBER 0 \[STATIC\]\_s*' ..
+  #  '2 STORE $0\_s*' ..
+  #  'x = o.o_var\_s*' ..
+  #  '3 LOAD arg\[-1\]\_s*' ..
+  #  '4 OBJ_MEMBER 0\_s*' ..
+  #  '5 STORE $0\_s*' ..
+  #  'x = o.s_var2\_s*' ..
+  #  '6 LOAD arg\[-1\]\_s*' ..
+  #  '7 OBJ_MEMBER 1 \[STATIC\]\_s*' ..
+  #  ' 8 STORE $0\_s*' ..
+  #  'x = o.o_var2\_s*' ..
+  #  '9 LOAD arg\[-1\]\_s*' ..
+  #  '10 OBJ_MEMBER 1\_s*' ..
+  #  '11 STORE $0\_s*' ..
+  #  '12 RETURN void',
+  #  g:instr2)
```

Comment:
Can this be replaced with another test?  If this is not needed, then this can be removed?

Reply:
Part of why this is draft, just to get overall feedback. If statics can not be accessed through an instance, then' there's a bunch of cleanup to do.

Reply:
After patch 9.0.1887, the static member variables and methods can be accessed only using the class name and cannot be
accessed using an instance.

Reply:
OK. I'll close this. I built this morning and see that `Found errors in Test_disassemble_interface_static_member():` is failing. I'll open a PR. (which may end up being redundant if it's in process elsewhere.

Reply:
I didn't notice this test failure.  I was running only the tests in the test_vim9_class.vim file.  I have the changes ready to fix this failure and will open a PR shortly.